lib/components/dashboard/degree-info/DistributionBarsJSX.tsx

```diff
@@ -199,25 +199,29 @@ const DistributionBarsJSX: FC<{ major: Major }> = ({ major }) => {
     // If double_count is string[], the specified distributions / fine requirements are 'whitelisted'
     // When a course satisfies a distribution, double_count takes the value of distribution.double_count
     if (!courseObj) return;
-    let distDoubleCount: string[] | undefined = undefined; // initial value
+    let distDoubleCount: string[] | undefined = ['All']; // double count by defualt
     reqs.forEach((reqGroup, i) => {
       let req = reqGroup[1][0]; // general distribution
       if (
-        (!distDoubleCount || distDoubleCount.includes(req.name)) &&
+        distDoubleCount &&
```

Comment:
distDoubleCount will never be undefined, false, 0, an empty string, or null. Thus, it will always compute to true and this check is extraneous and unnecessary.

Reply:
Or could it be undefined (ie. nonexistent). This makes sense if it could be nonexistent, but I think an empty array makes more sense as it's already checked by "distDoubleCount.includes(req.name)", saving one computation and cleaning up the logic. 

Basically, we make double_count required, and empty arrays means it can't double count with anything else.

Reply:
I think `distDoubleCount` could be undefined if the distribution does not have the field. As you said, it's optional atm.

logging `distDoubleCount`: 
<img width="709" alt="Screen Shot 2022-07-28 at 2 07 28 PM" src="https://user-images.githubusercontent.com/90944895/181425100-9a521a81-20d1-4b1c-a95f-20c62aff68aa.png">

Reply:
agreed. resolved.